lib/compiler/scjson-to-module.js

```diff
@@ -598,12 +654,18 @@ if(require.main === module){
       process.stdin.on('end',go);
     } else{
       var fs = require('fs');
-      jsonString = fs.readFileSync(input,'utf8'); 
+      jsonString = fs.readFileSync(input,'utf8');
       go();
     }
 }
 
 function go(){
-    var o = startTraversal(JSON.parse(jsonString),moduleType);
-    console.log(o.module); 
+    startTraversal(JSON.parse(jsonString), {debug: true}).then(
+        function resolved(jsModule) {
+            console.log(jsModule.module);
+        },
+        function rejected(err) {
+            console.error(err);
+        }
+    );
 }
```

Comment:
To summarize changes to this file:
* Provided async interface using ES6 generators
* Reduced whitespace and unnecessary characters in generated code to save memory an post-processing time
* Added 'use strict' option to generated code.